apps/verify/translations/src/en/fields.json

```diff
@@ -3,12 +3,12 @@
     "legend": "You can use a biometric residence permit (BRP) number or a unique application number (UAN) to sign in to this form.",
     "options": {
       "brp": {
-        "label": "Using a BRP number",
-        "hint": "For example, RZ1234567"
+        "label": "Using a biometric residence permit (BRP) number",
```

Comment:
The text you have changed to is from the older version of the Figma design I think. Could you confirm with UCD and  check we're not updating to an older version?